A job failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/19226317855/job/54954325921) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/54954325921) [(plain)](https://github.com/rust-lang/rust/commit/7bf478025eb4c339e4f792737e1f0e6194c24756/checks/54954325921/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  PR_CI_JOB: 1
  SKIP_SUBMODULES: src/gcc
  IMAGE: tidy
##[endgroup]
error: PRs changing the `nightly` channel should be sent to the `main` branch!
##[error]Process completed with exit code 1.
##[group]Run echo "disk usage:"
echo "disk usage:"
df -h
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
```

</details>

--- a/.github/workflows/post-merge.yml
+++ b/.github/workflows/post-merge.yml
@@ -6,7 +6,7 @@ name: Post merge analysis
 on:
   push:
     branches:
-      - master
+      - main
 
 jobs:
   analysis:

--- a/src/ci/scripts/verify-channel.sh
+++ b/src/ci/scripts/verify-channel.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 # We want to make sure all PRs are targeting the right branch when they're
 # opened, otherwise we risk (for example) to land a beta-specific change to the
-# master branch. This script ensures the branch of the PR matches the channel.
+# default branch. This script ensures the branch of the PR matches the channel.
 
 set -euo pipefail
 IFS=$'\n\t'
@@ -16,7 +16,7 @@ fi
 channel=$(cat "$(ciCheckoutPath)/src/ci/channel")
 case "${channel}" in
     nightly)
-        channel_branch="master"
+        channel_branch="main"
         ;;
     beta)
         channel_branch="beta"

--- a/src/stage0
+++ b/src/stage0
@@ -2,7 +2,7 @@ dist_server=https://static.rust-lang.org
 artifacts_server=https://ci-artifacts.rust-lang.org/rustc-builds
 artifacts_with_llvm_assertions_server=https://ci-artifacts.rust-lang.org/rustc-builds-alt
 git_merge_commit_email=bors@rust-lang.org
-nightly_branch=master
+nightly_branch=main
 
 # The configuration above this comment is editable, and can be changed
 # by forks of the repository if they have alternate values.

--- a/triagebot.toml
+++ b/triagebot.toml
@@ -1663,7 +1663,7 @@ labels = ["has-merge-commits", "S-waiting-on-author"]
 format = "rustc"
 project-name = "Rust"
 changelog-path = "RELEASES.md"
-changelog-branch = "master"
+changelog-branch = "main"
 
 [shortcut]
 

